docs/fidesops/docs/guides/saas_postprocessors.md

```diff
@@ -84,14 +85,14 @@ data = [
 ```
 
 Result:
-```
-result = [
-          {
-              "id": 1397429347
-              "email_contact": "somebody@email.com"
-              "name": "Somebody Awesome"
-          }
-        ]
+```json
+"result" = [
```

Comment:
If we're going to change this to json formatting, can we fix the json?

Reply:
Fixed!